ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -18,6 +18,15 @@
 
 ### FAN RUN CONTROL ###
 
+FAN_SS_VSC_WDPM:
```

Comment:
please add new GUID

Reply:
@cstirdivant - thanks for response - have added guid. Thanks, Ciaran